### PR TITLE
PhysicalMachineChaos: support network down

### PR DESCRIPTION
**Pull request by Wx-sy**
Signed-off-by: wangxu <wangxu_yewu@cmss.chinamobile.com>

### What problem does this PR solve?

minor update in PhysicalMachineChaos：add network-down

### What's changed and how it works?

changed in physical_machine_chaos and some yaml config files

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [x] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:
```
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by iguoyr** [comment]
@Wx-sy Hi, thanks for your contribution! Please fix DCO first, you could refer to [this document](https://github.com/chaos-mesh/chaos-mesh/pull/3125/checks?check_run_id=6003885759).

File: api/v1alpha1/physical_machine_chaos_types.go [hunk 1]
```diff
@@ -332,6 +337,13 @@ type NetworkPartitionSpec struct {
 	AcceptTCPFlags string `json:"accept-tcp-flags,omitempty"`
 }
 
+type NetworkDownSpec struct {
+	// the network interface to impact
+	Device string `json:"device,omitempty"`
+	// NIC down time, time units: ns, us (or µs), ms, s, m, h.
+	Duration string `json:"duration,omitempty"`
```
Comment (WangXiangUSTC) [inline api/v1alpha1/physical_machine_chaos_types.go:344]: There is another `Duration` in `PhysicalMachineChaosSpec `, I think it's better to resue this one
https://github.com/chaos-mesh/chaos-mesh/blob/8f10165d47dac0f917b4527bb4e8abbf0d0a6db3/api/v1alpha1/physical_machine_chaos_types.go#L78

**Review by WangXiangUSTC — COMMENTED** [review]
Thanks for your contribution!

File: api/v1alpha1/physical_machine_chaos_types.go (hunk 1 above)
Comment (WangXiangUSTC) [reply api/v1alpha1/physical_machine_chaos_types.go:344]: We can keep the `Duration` here, and set it to the value of `PhysicalMachineChaosSpec.Duration` in the webhook.

**Comment by FingerLeader** [comment]
@Wx-sy Could you please fix the ci and update the CHANGELOG.md?

**Comment by FingerLeader** [comment]
This feature has been merged in #3477, I'll close this PR.

**Pull request closed without merging**